raiden/ui/cli.py

```diff
@@ -655,7 +656,12 @@ def run_app(
     if sync_check:
         check_synced(blockchain_service)
 
-    database_path = os.path.join(datadir, 'netid_%s' % net_id, address_hex[:8], 'log.db')
+    database_path = os.path.join(
+        datadir,
+        'netid_%s' % net_id,
+        address_hex[:8],
+        'log_version_' + str(RAIDEN_DB_VERSION) + '.db',
```

Comment:
II'd put `f'v{RAIDEN_DB_VERSION}_log.db` here.

Reply:
I changed it.